Gordon360/Controllers/AccountsController.cs

```diff
@@ -118,6 +120,8 @@ public async Task<ActionResult<IEnumerable<BasicInfoViewModel>>> SearchWithSpace
             string? minor,
             string? hall,
             string? classType,
+            string? preferredClassYear,
+            string? preferredClassYearRange,
```

Comment:
```suggestion
            SearchYearRangeViewModel? preferredClassYearRange,
```

@EjPlatzer I have a personal preference of having this be a viewmodel along the lines of 
```
namespace Gordon360.Models.ViewModels
{
    public class SearchYearRange
    {
        int StartYear { get; set; }
        int EndYear { get; set; }
    }
}
```
Because right now the Integer Converting of splitting a string feels odd and clunky. What do you think?

Reply:
@antoniavonto @ArabellaJi 
If Evan is also on board, this would have to be a change that happens in the UI where you would need to pass a new JSON object into the `AdvancedSearchViewModel`